pkg/authenticator/authenticator.go

```diff
@@ -153,6 +154,7 @@ func (auth *Authenticator) Login() error {
 
 	// clean up the client cert so it's only available in memory
 	os.Remove(auth.Config.ClientCertPath)
+	log.InfoLogger.Printf(log.CAKC016I)
```

Comment:
Securitywise - Can't this help attacker to understand the timeline he can get a memory dump with the cert?

Reply:
//Clean (capital C)

Reply:
//Remove the client cert.....

Reply:
@shulifink thanks, but that is just a comment for developers. I am looking for feedback on the log messages.

Reply:
@shaharglazner do you see a security concern here? UX-wise there is value in this log message.